common/containers/Tabs/GenerateWallet/index.js

```diff
@@ -11,30 +14,66 @@ class GenerateWallet extends Component {
 
   static propTypes = {
     // state
-    generateWalletPassword: PropTypes.object,
-    showPassword: PropTypes.bool,
-    hasDownloadedWalletFile: PropTypes.bool,
-    generateWalletFile: PropTypes.bool,
-    canProceedToPaper: PropTypes.bool,
+    activeStep: PropTypes.string,
+    walletFile: PropTypes.object,
+    privateKey: PropTypes.string,
+    address: PropTypes.string,
     // actions
     showPasswordGenerateWallet: PropTypes.func,
-    generateFileGenerateWallet: PropTypes.func,
-    downloadFileGenerateWallet: PropTypes.func,
-    confirmContinueToPaperGenerateWallet: PropTypes.func
+    generateUTCGenerateWallet: PropTypes.func,
+    downloadUTCGenerateWallet: PropTypes.func,
+    confirmContinueToPaperGenerateWallet: PropTypes.func,
+    continueToUnlockWallet: PropTypes.func
   };
 
   render() {
-    return <GenerateWalletPasswordComponent {...this.props} />;
+    const { activeStep } = this.props;
+    let content;
+
+    switch (activeStep) {
+      case 'password':
+        content = <EnterPassword {...this.props} />;
+        break;
+
+      case 'download':
```

Comment:
Ah shite I see that's already been merged. I could file a new PR, or one of y'all could pull it in from over here: https://github.com/wbobeirne/MyEtherWallet/commit/f3fd71f777b079d2fa4793e809b054d529a559e5